main.go

```diff
@@ -400,3 +410,24 @@ func runScorecardSettings(githubEventName, scorecardPolicyFile, scorecardResults
 	}
 	return &result, nil
 }
+
+func signScorecardResult(scorecardResultsFile string) error {
+	os.Setenv("COSIGN_EXPERIMENTAL", "true")
+	ctx := context.Background()
+
+	// Sign the scorecard result data and generate certificate and tlog entry in Rekor.
+	keyOpts := sign.KeyOpts{
+		FulcioURL:    "https://fulcio.sigstore.dev",
+		RekorURL:     "https://rekor.sigstore.dev",
+		OIDCIssuer:   options.DefaultOIDCIssuerURL,
+		OIDCClientID: "sigstore",
+	}
+	regOpts := options.RegistryOptions{}
+
+	_, err := sign.SignBlobCmd(ctx, keyOpts, regOpts, scorecardResultsFile, true, "", "", time.Minute)
+	if err != nil {
```

Comment:
I'm curious: Where is this function defined? I only managed to find but the signature is different https://github.com/sigstore/cosign/blob/main/cmd/cosign/cli/sign/sign_blob.go#L56 :/

Reply:
This one, I think it's from a different release
https://github.com/sigstore/cosign/blob/v1.5.2/cmd/cosign/cli/sign/sign_blob.go#L58